src/modules/Map/services/mapUtils.js

```diff
@@ -83,94 +85,119 @@ export const checkContraints = ({
   }, false);
 };
 
-let listenerWaiter = 0;
+export function getInteractionsOnEvent ({
+  eventType,
+  map,
+  point,
+  interactions: eventInteractions,
+  features = map.queryRenderedFeatures(point),
+}) {
+  let interactions = false;
```

Comment:
This default value let the reader thinks that `interactions` is a boolean value. Can it be a `null` (or undefined ) instead?
```suggestion
  let interactions = null;
```

Reply:
I don't really want to refacto everything in this file. That's not a part of code which take parts of this PR but some code which have already been reviewed and merged weeks ago.